sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -505,6 +508,7 @@ def query_items_change_feed(
             kwargs['priority'] = priority
         feed_options = _build_options(kwargs)
         feed_options["isStartFromBeginning"] = is_start_from_beginning
+        feed_options["startTime"] = start_time
```

Comment:
What if the customer does not pass the time in the correct format? Also, is there a way to validate that the time is specified in UTC Timezone? The backend expects UTC

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/dfee5ad2b0363b9c4209a028e41fc486ba7a309e/Microsoft.Azure.Cosmos/src/ChangeFeed/ChangeFeedStartFromTime.cs#L24-L27

Reply:
I added tests for that, and according to documentation if that were to happen then start time will simply be ignored as an option by backend.

Reply:
That's the reason we throw an exception in .NET. having the backend silently ignore it is not a good user experience because the user has certain expectations and they cannot understand why the behavior does not match. If we can detect the case, I would vote for erroring out

Reply:
We should support a datetime object with a local timezone, if a timezone is specified then it's easy to convert to UTC before sending over the wire. Though we can add a note to the docstring that if no timezone is specified, then the object is assumed to be in UTC and will be treated as such.